elpy-django.el

```diff
@@ -168,27 +168,43 @@ test arguments in `elpy-django-test-runner-args'."
     (setq dj-commands-str (mapcar (lambda (x) (s-trim x)) dj-commands-str))
     (sort dj-commands-str 'string-lessp)))
 
+
+(defvar elpy-django--test-runner-cache '()
```

Comment:
Make this `nil` not `'()` – this is not Scheme :-)

Reply:
Fixed